swagger_spec_validator/validator20.py

```diff
@@ -69,6 +70,8 @@ def validate_spec(spec_dict, spec_url='', http_handlers=None):
         http client built into jsonschema's RefResolver is used. This
         is a mapping from uri scheme to a callable that takes a
         uri.
+    :param schema_pkg: package of the json schema file.
+    :param schema_path: package relative path of the json schema file.
```

Comment:
These don't exist anymore. Please instead add a line documenting what `schema_dict` is supposed to be.